test/e2e/memtierd.test-suite/no-policy/n4c16/test01-mover/code.var.sh

```diff
@@ -0,0 +1,28 @@
+memtierd-setup
+
+# Test that 700M+ is moved to numa node {0..3} from a process
+MEME_CGROUP=e2e-meme MEME_BS=700M MEME_BWC=1 MEME_BWS=700M memtierd-meme-start
+
+match-moved() {
+    local moved_regexp=$1
+    local target_numa_node=$2
+    if ( memtierd-command "stats -t move_pages -f csv | awk -F, \"{print \\\$6}\""; grep ${moved_regexp} <<< $COMMAND_OUTPUT); then
+        echo "memtierd moved expected memory to the target numa node ${target_numa_node}"
+    else
+        error "error: memtierd did not move expected amount of memory!"
+    fi
+}
+
+for target_numa_node in {0..3}
+do
+  MEMTIERD_YAML=""
+  memtierd-start
+  memtierd-command "pages -pid ${MEME_PID}"
+  memtierd-command "mover -pages-to ${target_numa_node}"
+  sleep 10
```

Comment:
Let's print the "waiting" text before real waiting starts.

Then let's stop waiting as soon as possible. I'd suggest polling every second if expected amount of memory has been moved. I think something like that is done in swapping out from the heat policy. With this I wish to run e2e tests as fast as possible...